### DISCUSSION TITLE
Is there a way to use c2v now?

### DISCUSSION THREAD

**Discussion by grd** [vlang/v#13350] [General]
And if not, when is the release of V0.3?

**Comment by JalonSolov** [comment]
No, you c2v is not released, yet, and there are no specific dates for 0.3 release of V.

**Reply by grd** [reply]
So there is no way to use c2v?

**Reply by JalonSolov** [reply]
Not until it is released.

**Reply by grd** [reply]
Hmmm... Have you worked with c2v? Do you know whether it is working?
